src/Sylius/Bundle/CoreBundle/spec/Sitemap/Factory/SitemapUrlFactorySpec.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+namespace spec\Sylius\Bundle\CoreBundle\Sitemap\Factory;
+ 
+use PhpSpec\ObjectBehavior;
+use Sylius\Bundle\CoreBundle\Sitemap\Model\SitemapUrlInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class SitemapUrlFactorySpec extends ObjectBehavior
+{
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Bundle\CoreBundle\Sitemap\Factory\SitemapUrlFactory');
+    }
+
+    function it_implements_sitemap_url_factory_interface()
+    {
+        $this->shouldImplement('Sylius\Bundle\CoreBundle\Sitemap\Factory\SitemapUrlFactoryInterface');
+    }
+
+    function it_creates_empty_sitemap_url(SitemapUrlInterface $sitemapUrl)
+    {
+        $sitemapUrl->getLastModification()->willReturn(null);
+        $sitemapUrl->getLocalization()->willReturn(null);
+        $sitemapUrl->getPriority()->willReturn(null);
+        $sitemapUrl->getChangeFrequency()->willReturn(null);
+
+        $this->createEmpty()->shouldBeSameAs($sitemapUrl);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getMatchers()
+    {
+        return [
+            'beSameAs' => function ($subject, $key) {
+                if (!$subject instanceof SitemapUrlInterface || !$key instanceof SitemapUrlInterface) {
+                    return false;
+                }
+
+                return ($subject->getChangeFrequency() === $key->getChangeFrequency()
+                    && $subject->getLocalization() === $key->getLocalization()
+                    && $subject->getLastModification() === $key->getLastModification()
+                    && $subject->getPriority() === $key->getPriority());
```

Comment:
Semicolon should be put in a new line. Whole return statement doesn't require brackets surrounding